src/SpecFlow.NetCore/AppConfig.cs

```diff
@@ -52,16 +54,34 @@ public static AppConfig CreateIn(DirectoryInfo directory)
 			return config;
 		}
 
-		public void Validate()
+		private static string GuessUnitTestProvider(DirectoryInfo directory)
```

Comment:
Yeah you have a good point, but you can always force provider by changing the generated app config. Currently we take in a path argument, do we still need to support that?